qemu/tests/cfg/hv_avic.cfg

```diff
@@ -0,0 +1,11 @@
+- hv_avic:
+    type = hv_avic
+    only Linux
+    required_qemu = [6.2.0,)
+    cpu_model_flags += ",hv_avic"
+    cpuid_pkg = "cpuid-20170122-5.el8+5.x86_64.rpm"
```

Comment:
We don't support and agree with that expose the detail rpm package which maybe contain some sensitive information to the upstream.
So please use the internal parameter instead of it.